components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -279,7 +279,7 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
                             <Check className="m-0.5 w-5 h-5" />
                             <div className="flex flex-col">
                                 <span>
-                                    36¢ for 10 credits or 1 hour of Standard workspace usage.{" "}
+                                    $0.36 for 10 credits or 1 hour of Standard workspace usage.{" "}
```

Comment:
Please make the currency dynamic (based on user region -- later we could improve it to actual subscription currency):

```suggestion
                                    {currency === "EUR" ? "€" : "$"}0.36 for 10 credits or 1 hour of Standard workspace usage.{" "}
```